app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="generalContainer">
+  <div>
+    <%= link_to "Back to recipes", recipes_path, class: "backButton" %>
+  </div>
+
+  <h1 class="pageTitle"> <%= capitalize_text(@recipe.name) %></h1>
+  <div class="recipeShowGeneralInfo">
+    <p class="preparationTime"><strong>Preparation Time: </strong> <%= @recipe.preparation_time_minutes %></p>
+    <%= render 'shared/change_public', recipe: @recipe %>
+    <p class="cookingTime"><strong>Cooking Time: </strong> <%= @recipe.cooking_time_minutes %></p>
+    <p class="recipeDescription"><%= @recipe.description %></p>
+    <%if recipe_author?(@recipe.user)%>
+      <button ><%= link_to "Generate Shopping List", shopping_list_path %></button>
+      <button id="addIngredientButton">Add Ingredient</button>
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20the%20user%20is%20the%20owner%20of%20the%20recipe%2C%20should%20lead%20to%20the%20form%20that%20allows%20the%20user%20to%20add%20new%20food.), this button should lead to the form to create a new food if the user is the owner of the recipe.